int16 direction
---
int16 vector